two_factor/forms.py

```diff
@@ -14,14 +14,16 @@
 
 class MethodForm(forms.Form):
     method = forms.ChoiceField(label=_("Method"),
-                               initial='generator',
                                widget=forms.RadioSelect)
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.fields['method'].choices = [
+
+        method = self.fields['method']
+        method.choices = [
             (m.code, m.verbose_name) for m in registry.get_methods()
         ]
+        method.initial = method.choices[0][0]
```

Comment:
ok, i went and double checked. It's needed, otherwise there's no default.

This isn't a bad thing per se but I didn't want to change current behaviour.